src/server.rs

```diff
@@ -674,13 +686,28 @@ impl Server {
     pub fn mark_dirty(&mut self) {
         self.needs_cleanup = true;
     }
+
+    pub fn mirror_send(&mut self, bytes: &BytesMut) {
+        match self.mirror_manager.as_mut() {
+            Some(manager) => manager.send(bytes),
```

Comment:
Mirroring should be best effort imo, this will block if the mirror channel buffer is full because the mirror can't absorb any more traffic, I think?

Reply:
Yes, mirroring is best-effort. `manager.send` uses `try_send` under the hood. The reason I have a `manager` is to handle the fanout because we could have one `Server` matched with more than one `MirroredClient`
https://github.com/levkk/pgcat/pull/341/files#diff-453e032b0a6294b617b502297ffd4ffcce47d4f81e06f16b602c0d6f89afebb9R215-R225